docs/src/main/sphinx/develop/group-provider.md

```diff
@@ -20,18 +20,22 @@ must be wrapped as a Trino plugin and installed on the cluster.
 
 After a plugin that implements `GroupProviderFactory` has been installed on the coordinator,
 it is configured using an `etc/group-provider.properties` file.
-All the properties other than `group-provider.name` are specific to
+All the properties other than `group-provider.name` and `group-provider.group-case` are specific to
 the `GroupProviderFactory` implementation.
 
-The `group-provider.name` property is used by Trino to find a registered
+* The `group-provider.name` property is used by Trino to find a registered
 `GroupProviderFactory` based on the name returned by `GroupProviderFactory.getName()`.
-The remaining properties are passed as a map to
+* The `group-provider.group-case` property can be used to transform the case of groups. Allowed values
+are `[keep, upper, lower]`. This property is not mandatory and the default is `keep`, which does not
```

Comment:
Let's also explain what `upper` and `lower` does. It should be already obvious, but I am under impression that `group-provider.group-case` is not super self explanatory and saying that it will result in upper casing and lower casing could be helpful.

Or maybe it is because I am not a native speaker...